### ISSUE TITLE
Add `member_joined_channel` event

### ISSUE THREAD

**Issue by fkrauthan** [abdolence/slack-morphism-rust#175]
It would be great to have support for `member_joined_channel` event (https://api.slack.com/events/member_joined_channel).

Currently the error is:

```
json_error: Error("unknown variant `member_joined_channel`, expected one of `message`, `app_home_opened`, `app_mention`, `app_uninstalled`, `link_shared`, `emoji_changed`", line: 0, column: 0),
        json_body: Some(
            "{\"envelope_id\":\"f75fbe8a-a4eb-4666-***-439f0986c1fe\",\"payload\":{\"token\":\"***\",\"team_id\":\"T04N165GK0T\",\"context_team_id\":\"***\",\"context_enterprise_id\":null,\"api_app_id\":\"***\",\"event\":{\"type\":\"member_joined_channel\",\"user\":\"U04NDV8606M\",\"channel\":\"C04NR0SQBRN\",\"channel_type\":\"C\",\"team\":\"***\",\"inviter\":\"U04NR0GHBGQ\",\"event_ts\":\"1677093519.000500\"},\"type\":\"event_callback\",\"event_id\":\"Ev04QDN07G23\",\"event_time\":1677093519,\"authorizations\":[{\"enterprise_id\":null,\"team_id\":\"***\",\"user_id\":\"U04NDV8606M\",\"is_bot\":true,\"is_enterprise_install\":false}],\"is_ext_shared_channel\":false,\"event_context\":\"4-***\"},\"type\":\"events_api\",\"accepts_response_payload\":false,\"retry_attempt\":0,\"retry_reason\":\"\"}",
        ),
```

or is there currently a way to add support for additional types myself?

**Comment by abdolence** [comment]
Yes, we still have missing models and fields in some places.
Thanks for the PR, this helps everyone who is using the library!

**Comment by fkrauthan** [comment]
Thanks for merging. Any idea when you might be pushing the next release with the changes?

**Comment by abdolence** [comment]
In a few days, until then you can just use a reference to the git:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories

**Comment by abdolence** [comment]
This is released now in v1.7.0.
Sorry for the delay, I was waiting for another PR, but it is not finalised yet for now.

**Comment by fkrauthan** [comment]
No worries. I totally forgot that I can pin the repo as my dependency source. Which works perfectly for now.
